src/charm.py

```diff
@@ -793,6 +793,4 @@ def server_ca_cert_path(self) -> Optional[str]:
 
 
 if __name__ == "__main__":
-    # We need use_juju_for_storage=True because ingress_per_unit
-    # requires it to keep track of the ingress address.
-    main(LokiOperatorCharm, use_juju_for_storage=True)
```

Comment:
@PietroPasotti @dstathis Our other IPU charm, prometheus, doesn't use controller storage.
And in ingress_per_unit v1, the only use of stored state is for current_urls, which seems like we don't need to rely on controller storage. Wdyt?

Reply:
I'd tend to agree. worst that can happen is you get two ready events instead of one, and given the hot reloading this won't even trigger a restart.